samples/nsxt_change_vm_vif.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+"""
+Written by Yasen Simeonov
+Github: https://github.com/yasensim
+
+This code is released under the terms of the Apache 2
+http://www.apache.org/licenses/LICENSE-2.0.html
+
+Example script to change the network of the Virtual Machine NIC
+that includes NSX-T opeque switch
+
+"""
+
+import atexit
+import ssl
+
+from tools import cli
+from tools import tasks
+from pyVim import connect
+from pyVmomi import vim, vmodl
+
+
+def get_obj(content, vimtype, name):
+    """
+     Get the vsphere object associated with a given text name
+    """
+    obj = None
+    container = content.viewManager.CreateContainerView(content.rootFolder,
+                                                        vimtype, True)
+    for view in container.view:
+        if view.name == name:
+            obj = view
+            break
+    return obj
+
+
+def get_args():
```

Comment:
Please move this method above get_obj(), just to make sure to follow Standard sample template followed by all samples